<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MosaicNSync</class>
 <widget class="QDialog" name="MosaicNSync">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>977</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>977</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>977</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Constantia</family>
    <pointsize>13</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>모씽 v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/icons8-webcam-96.png</normaloff>icons/icons8-webcam-96.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(140, 140, 140)rgb(61, 61, 61);
font: 13pt &quot;Constantia&quot;;

</string>
  </property>
  <widget class="QDialogButtonBox" name="dialog_ok">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>520</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>931</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Infomation</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textLog">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>861</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>531</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Constantia</family>
     <pointsize>13</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>mosync</string>
   </property>
   <widget class="QPushButton" name="btn5_disableMosaic">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>  Disable
  Mosaic</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-virtual-reality-96.png</normaloff>icons/icons8-virtual-reality-96.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QCheckBox" name="chk1_sync">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>체크박스입니다</string>
    </property>
    <property name="text">
     <string>continueSync</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn1_mosaic">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>EnableMosaic</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-laptop-96.png</normaloff>icons/icons8-laptop-96.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4_sync">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Enable_Sysnc</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-audio-cable-96.png</normaloff>icons/icons8-audio-cable-96.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_master">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Master</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Slave</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="combo_hz">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <item>
     <property name="text">
      <string>24hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>60hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>120hz</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>110</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Profile3d global setting</string>
   </property>
   <widget class="QPushButton" name="btn8_setAdvanced">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Workstation App - Advanced Streaming</string>
    </property>
    <property name="text">
     <string>Advanced</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-hdd-96.png</normaloff>icons/icons8-hdd-96.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn7_setDefault">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Default</string>
    </property>
    <property name="text">
     <string>Default</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-usb-memory-stick-96.png</normaloff>icons/icons8-usb-memory-stick-96.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>230</y>
     <width>311</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>ect</string>
   </property>
   <widget class="QPushButton" name="btn6_openNvcpl">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>nvidia 제어판을 엽니다.</string>
    </property>
    <property name="text">
     <string>OpenNvcplui</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-video-card-96.png</normaloff>icons/icons8-video-card-96.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2_clear">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>81</width>
      <height>43</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>43</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>43</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Log clear</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-keyboard-96.png</normaloff>icons/icons8-keyboard-96.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3_current">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>151</width>
      <height>43</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>43</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>43</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>현제 상태를 체크 합니다.</string>
    </property>
    <property name="text">
     <string>checkCurrent</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>icons/icons8-tape-drive-96.png</normaloff>icons/icons8-tape-drive-96.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialog_ok</sender>
   <signal>accepted()</signal>
   <receiver>MosaicNSync</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_ok</sender>
   <signal>rejected()</signal>
   <receiver>MosaicNSync</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
